docs/en/pid-eaa-issuance.rst

```diff
@@ -602,9 +655,6 @@ A DPoP-bound Access Token is provided by the PID Provider Token endpoint as a re
   * - **jti** 
     - It MUST be a String in *uuid4* format. Unique Token ID identifier that the RP MAY use to prevent reuse by rejecting the Token ID if already processed.
     - [:rfc:`9068`], [:rfc:`7519`].
-  * - **nonce** 
-    - It MUST be a random string of at least 32 alphanumeric characters. The value type of this claim MUST be a string, where the value is a **c_nonce** provided by the PID Provider.
-    - [`OpenID.Core#AuthRequest <https://openid.net/specs/openid-connect-core-1_0.html#AuthRequest>`_].
```

Comment:
I'm lost. Where is the nonce now?

Reply:
We don't have nonce in [rfc9068](https://datatracker.ietf.org/doc/html/rfc9068), so the nonce parameter here is not compliant with RFC. Moreover, we actually don't need it because the token request is right after the auth req where we have PKCE.

Reply:
All right!